code/modules/mob/living/living.dm

```diff
@@ -962,7 +962,7 @@ Sorry Giacom. Please don't be mad :(
 			src << "<span class='notice'>You're too exhausted to keep going...</span>"
 			Weaken(5)
 			setStaminaLoss(health - 2)
-		update_health_hud()
+	update_health_hud()

```

Comment:
This is so our health hud icon is updated when adjustStaminaLoss(-1) is called and makes staminaloss null.